{"stats": {"errors": 1, "failures": 0, "passes": 0, "skipped": 0, "encoding": "UTF-8", "total": 1}, "results": [{"classname": "index_test:TxTest", "name": "test_sign_input", "time": 0.07223725318908691, "type": "error", "errtype": "builtins.AttributeError", "message": "'Tx' object has no attribute 'sign_hash'", "tb": "Traceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n  File \"/home/jonastesterelli/tx-signing-inputs-jsong-pb-sf-072618/test/index_test.py\", line 29, in test_sign_input\n    self.assertTrue(tx.sign_input(0, private_key, SIGHASH_ALL))\n  File \"/home/jonastesterelli/tx-signing-inputs-jsong-pb-sf-072618/index.ipynb\", line 28, in sign_input\n    \"from helper import (\\n\",\nAttributeError: 'Tx' object has no attribute 'sign_hash'\n"}]}